src/storekit.cs

```diff
@@ -450,7 +480,7 @@ interface SKProductsRequestDelegate {
 		void ReceivedResponse (SKProductsRequest request, SKProductsResponse response);
 	}
 
-#if !MONOMAC
+#if !MONOMAC && !WATCH
```

Comment:
`!WATCH` b/c `UIViewController` is not available on Watch.

Reply:
Or is there a preference for doing this w/ macros vs at the top of the file?

Reply:
whatever is the simplest / shortest
sometimes you only need one `#if` but a dozen `using`
other times you can avoid a dozen `#if` with a single `using`